path_finding.asciidoc

```diff
@@ -321,6 +351,10 @@ Therefore, Selena's node must consider how long to keep this knowledge before as
 [[stuck_payments]]
 ===== Stuck payments
 
+In the current implementation of the Lightning Network there is a possibility that a payment attempt becomes "stuck", neither fulfilled nor cancelled by an error. This can happen due to a bug on an intermediary node, a node going offline while handling HTLCs, or a malicious node holding HTLCs without reporting an error. In all of these cases, the HTLC cannot be resolved until it expires. The timelock (CLTV) that is set on every HTLC helps resolve this condition (among other possible HTLC routing and channel failures). However, this means that the sender of the HTLC has to wait until expiry, and the funds committed to that HTLC remain unavailable until the HTLC expires. Furthermore, the sender *cannot retry* that same payment, because if they do they run the risk of *both* the original and the retried payment succeeding - the recipient gets paid twice. This is because once sent an HTLC cannot be "cancelled" by the sender - it either has to fail or expire. Stuck payments, while rare, create an unwanted user experience, where the user's wallet cannot pay or cancel a payment.
+
+One proposed solution to this problem is called "stuckless payments", and it depends on Point Time-Locked Contracts (PTLCs), which are payment contract that use a different cryptographic primitive than HTLCs (i.e. point addition on the elliptic curve instead of a hash and secret pre-image).
```

Comment:
I think it is important to note that 

1.) PTLCs basically require taproot. While there is the scriptless scripts way of doing this with ECDSA magic only with taproot and the resulting adaptor signatures we will go down this way 
2.) stuckless payments have there own issues / challanges the most important one is that you can decide to not settle an onion and thus cancle it. how ever the liquidity bound in that onion is still locked until the stuck payment returns. So in the worst case there will be a lot of bound liquidity

overall I am not sure if the note about stuckless payments should be moved to the HTLC / Routing chapter as this is not so much about path finding but more about transport of onions / HTLCs